devices/Ld2410/Ld2410/Reporting/EngineeringModeReportFrame.cs

```diff
@@ -0,0 +1,122 @@
+﻿namespace Ld2410.Reporting
+{
+	/// <summary>
+	/// Represents the engineering more report frame produced by the radar when engineering mode is enabled.
+	/// </summary>
+	public sealed class EngineeringModeReportFrame : BasicReportFrame
+	{
+		/// <summary>
+		/// Gets the maximum distance gate supported by the radar for moving targets.
+		/// </summary>
+		public byte MaxMovingDistanceGate { get; internal set; }
+
+		/// <summary>
+		/// Gets the maximum distance gate supported by the radar for static targets.
+		/// </summary>
+		public byte MaxStaticDistanceGate { get; internal set; }
+
+		/// <summary>
+		/// Gets the movement energy at distance gate 0.
+		/// </summary>
+		public byte Gate0MovementEnergy { get; internal set; }
```

Comment:
I really do feel a table will be better. If things goes in pair, then maybe even a class and an array of it.

Reply:
This is done.